devices/HumanDataCollector/HumanDataCollector.cpp

```diff
@@ -551,6 +851,45 @@ bool HumanDataCollector::detach()
         stop();
     }
 
+    if (pImpl->matioLogger) {
+
+        // Set the string variables elements as cell arrays to mat cell variables
+        writeVectorOfStringToMatCell("stateJointNames", pImpl->humanDataStruct.stateJointNames, pImpl->matFilePtr);
+        writeVectorOfStringToMatCell("wrenchMeasurementSourceNames", pImpl->humanDataStruct.wrenchMeasurementSourceNames, pImpl->matFilePtr);
+        writeVectorOfStringToMatCell("wrenchEstimateSourceNames", pImpl->humanDataStruct.wrenchEstimateSourceNames, pImpl->matFilePtr);
+        writeVectorOfStringToMatCell("dynamicsJointNames", pImpl->humanDataStruct.dynamicsJointNames, pImpl->matFilePtr);
+
+        // Correct the time values stored in the time vector to zero start value
+        std::transform( pImpl->humanDataStruct.time.begin(), pImpl->humanDataStruct.time.end(), pImpl->humanDataStruct.time.begin(), std::bind2nd( std::plus<long>(), - pImpl->humanDataStruct.time.at(0) ) );
```

Comment:
How could this super long line trick clang format?

Reply:
I am not certain if the clang is correctly setup, I will ensure it. For the time being https://github.com/robotology/human-dynamics-estimation/pull/181/commits/9c24441031952724777a1947969b079deff49c14